src/backend/launcher.ts

```diff
@@ -211,6 +211,10 @@ async function prepareLaunch(
         }
       }
 
+      gameScopeCommand.push(
+        ...shlex.split(gameSettings.gamescope.additionalOptions)
+      )
```

Comment:
You have to guard this, `additionalOptions` might be undefined (the types around the config system are a little unsafe)
```suggestion
      if (gameSettings.gamescope.additionalOptions)
        gameScopeCommand.push(
          ...shlex.split(gameSettings.gamescope.additionalOptions)
        )
```
I don't think this lints, don't merge as-is